src/components/forms/pool_actions/MigrateForm/components/MigratePreviewModal/MigratePreviewModal.vue

```diff
@@ -0,0 +1,121 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+import { FullPool } from '@/services/balancer/subgraph/types';
+
+import { MigrateMathResponse } from '../../composables/useMigrateMath';
+
+import MigratePoolRisks from './components/MigratePoolRisks.vue';
+import MigratePoolsInfo from './components/MigratePoolsInfo.vue';
+import MigrateActions from './components/MigrateActions.vue';
+import InvestSummary from '../../../InvestForm/components/InvestPreviewModal/components/InvestSummary.vue';
+
+import { TokenInfo } from '@/types/TokenList';
+
+import { PoolMigrationInfo } from '../../types';
+
+/**
+ * TYPES
+ */
+type Props = {
+  poolMigrationInfo: PoolMigrationInfo;
+  fromPool: FullPool;
+  toPool: FullPool;
+  fromPoolTokenInfo: TokenInfo;
+  toPoolTokenInfo: TokenInfo;
+  math: MigrateMathResponse;
+};
+
+/**
+ * PROPS & EMITS
+ */
+const props = defineProps<Props>();
+
+const priceImpact = ref(props.math.priceImpact.value);
+const fiatTotalLabel = ref(props.math.fiatTotalLabel.value);
+const fiatTotalWithPriceImpact = ref(
+  props.math.fiatTotalWithPriceImpactLabel.value
+);
+const fiatTotal = ref(props.math.fiatTotal.value);
```

Comment:
This makes these "static" because if you use `toRefs` it seems like it's behaving like "computed" and you would have all the values wiped out when the migration is complete.

This is so I won't have to show 0's...